scripts/setup/bootstrap.sh

```diff
@@ -68,6 +68,12 @@ _bootstrap_or_activate() {
     # Strip off the 'scripts[/setup]' directory, leaving the root of the repo.
     _CHIP_ROOT="$(cd "${_BOOTSTRAP_DIR%/setup}/.." && pwd)"
 
+    local _PIGWEED_CIPD_JSON_FILE="$_CHIP_ROOT/third_party/pigweed/repo/pw_env_setup/py/pw_env_setup/cipd_setup/pigweed.json"
+    # TODO need to find a better location then /tmp/test.json
+    local _GENERATED_PIGWEED_CIPD_JSON_FILE="/tmp/test.json"
```

Comment:
Where might be a good location to put this newly generated pigweed.json file?

Reply:
we could also place it somewhere in pw_environment_root and named uniquely (like matter_sdk_cipd.json) since they are very related.

Reply:
It seems like we are semi tied to needing the name to be `pigweed.json` as there are hardcoded paths in various places that seem to look for `.environment/cipd/packages/<name_of_json_file>` where `name_of_json_file` is the filename without json extension. I could clean this up if we think that there is value. But as of right now there are no pigweed.json files in the `.environment` directory, so we should be okay with just using that name

https://github.com/project-chip/connectedhomeip/actions/runs/6643593250/job/18050974786?pr=29968